lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -80,7 +88,7 @@ export type LoggerOptions = {
 };
 
 /**
- * Telemetry info about library
+ * Use this to configure the library-specific options
```

Comment:
Do we want this to be called out? This won't be set by users but rather by the consuming library.

Reply:
Makes sense. Changed.